src/components/CalendarBody.tsx

```diff
@@ -54,6 +55,9 @@ interface CalendarBodyProps<T extends ICalendarEventBase> {
   headerComponent?: React.ReactElement | null
   headerComponentStyle?: ViewStyle
   hourStyle?: TextStyle
+  minTime?: string
+  maxTime?: string
+  steps?: number
```

Comment:
```suggestion
  minTimeMinutes?: number
  maxTimeMinutes?: number
  stepMinutes?: number
```

These props should be number(in minutes) for consistency with`scrollOffsetMinutes`.
Performance should be better because no loop and parsing happens.